### DISCUSSION TITLE
Not Being Able to Submit My academic status proof

### DISCUSSION THREAD

**Discussion by kypi03** [community/community#65118] [GitHub Education]
### Hi everyone,

Hi GitHub Education team,

I'm writing to you because I'm having trouble applying for the Student Developer Pack. I've tried uploading a picture of my student ID front and back, but the page just refreshes and I don't get any confirmation or email that my application is being reviewed.

I've checked the requirements and I'm sure that I'm eligible. I'm a current student at University of Tehran and I have a school-issued email address. I've also tried uploading the picture in different alignments (landscape and portrait), but that didn't seem to make a difference.

I'm not sure what I'm doing wrong. Could you please help me troubleshoot this issue?

Thank you for your time and consideration.

**Comment by arafajs** [comment]
Unfortunately, the GitHub Student Developer Pack is not available in Iran. GitHub has not specified why the pack is not available in Iran, but it is likely due to sanctions imposed by the United States government.

**Reply by kypi03** [reply]
my friends got it like a month ago and it works just fine.

**Comment by ApocalixDeLuque** [comment]
I have two friends with the same problem, we're from mexico and have the error where they try to upload the proof and just get redirected, i could get the student benefits tho, which is wierd, it seems to be a recent issue.

has anyone resolve this yet?

**Comment by nqlp** [comment]
What worked for me is taking a photo with my phone and uploading it. The most important thing is the date because it indicates your current school enrollment status. See #64963.

**Comment by pepper-lees** [comment]
I am getting the exact same issue, and I'm in the USA. This is really frustrating because it's not giving any feedback on what I'm doing wrong.
